### PR TITLE
Add details about components and bundles

### PR DESCRIPTION
**Pull request by nidup**

### CODE REVIEW

File: reference/component-or-bundle.rst.inc [hunk 1]
```diff
@@ -5,4 +5,8 @@ Since version 1.3, Akeneo PIM introduced components, that contain business logic
 
 We use different namespaces, Akeneo for the technical parts not coupled with the PIM, Pim for the Community Edition parts and Enterprise postfix for the Enterprise Edition parts.
 
+We organise the code as above in any new development and trying to update legagy code as much as we can in each minor version to make the stack more and more understandable.
+
 .. image:: /reference/component-or-bundle.png
+    :width: 400pt
+.
```
Comment (solivier) [inline reference/component-or-bundle.rst.inc]: new blanc line missing

File: reference/bundles/akeneo_bundles.rst.inc [hunk 2]
```diff
@@ -1,9 +1,19 @@
 .. _BatchBundle: https://github.com/akeneo/pim-community-dev/tree/master/src/Akeneo/Bundle/BatchBundle
+.. _ClassificationBundle: https://github.com/akeneo/pim-community-dev/tree/master/src/Akeneo/Bundle/ClassficationBundle
```
Comment (solivier) [inline reference/bundles/akeneo_bundles.rst.inc]: I got a 404 for this link

File: reference/components/akeneo_components.rst.inc [hunk 3]
```diff
@@ -1,18 +1,31 @@
 .. _Analytics: https://github.com/akeneo/pim-community-dev/tree/master/src/Akeneo/Component/Analytics
 .. _Batch: https://github.com/akeneo/pim-community-dev/tree/master/src/Akeneo/Component/Batch
+.. _Buffer: https://github.com/akeneo/pim-community-dev/tree/master/src/Akeneo/Component/Buffer
 .. _Classification: https://github.com/akeneo/pim-community-dev/tree/master/src/Akeneo/Component/Classification
 .. _Console: https://github.com/akeneo/pim-community-dev/tree/master/src/Akeneo/Component/Console
 .. _FileStorage: https://github.com/akeneo/pim-community-dev/tree/master/src/Akeneo/Component/FileStorage
+.. _Localization: https://github.com/akeneo/pim-community-dev/tree/master/src/Akeneo/Component/Localisation
```
Comment (solivier) [inline reference/components/akeneo_components.rst.inc]: I got a 404 too

File: reference/bundles/akeneo_bundles.rst.inc (hunk 2 above)
Comment (nidup) [inline reference/bundles/akeneo_bundles.rst.inc]: thx fixed

File: reference/components/akeneo_components.rst.inc (hunk 3 above)
Comment (skeleton) [inline reference/components/akeneo_components.rst.inc]: https://github.com/akeneo/pim-community-dev/tree/master/src/Akeneo/Component/Localization

File: reference/component-or-bundle.rst.inc (hunk 1 above)
Comment (nidup) [inline reference/component-or-bundle.rst.inc]: fixed

**Pull request merged**